src/main/java/com/connexta/security/markings/banner/BannerCreator.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.security.markings.banner;
+
+import static org.springframework.util.StringUtils.isEmpty;
+
+import com.connexta.security.markings.api.rest.models.ISM;
+import com.connexta.security.markings.util.InputValidator;
+import com.google.common.collect.ImmutableList;
+import java.util.List;
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class BannerCreator {
+  private static BannerCreator instance;
+
+  public static final String DELIMITER = "//";
+  public static final String SUB_DELIMITER = "/";
+  public static final String SPACE = " ";
+  private static List<String> SPECIAL_CLASSIFICATIONS =
+      ImmutableList.of("NU", "NR", "NC", "NS", "CTS");
+
+  // singleton - make the default constructor private
+  private BannerCreator() {}
+
+  public static BannerCreator getInstance() {
+    if (instance == null) {
+      instance = new BannerCreator();
+    }
+    return instance;
+  }
```

Comment:
❓ Since you're using Spring, could you not just create a bean which effectively does the same thing of insuring there is only one instance

Reply:
f thats a great idea. doing now